ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/BankAccountTest.java

```diff
@@ -141,8 +141,8 @@ public void testAddTransferMethod_displaysFeeElementsOnTmcResponse() {
                 matches(withText(R.string.add_transfer_method_fee_label)));
         onView(withId(R.id.add_transfer_method_processing_label)).check(
                 matches(withText(R.string.add_transfer_method_processing_time_label)));
-        onView(withId(R.id.add_transfer_method_fee_value)).check(matches(withText("USD 2.00")));
```

Comment:
Leave the data the same please

Reply:
@skoong returned, but I unable to to handle processing time. Where I should place it?